knative-operator/pkg/controller/knativeserving/kourier/kourier.go

```diff
@@ -124,3 +129,22 @@ func replaceImageFromEnvironment(prefix string, scheme *runtime.Scheme) mf.Trans
 		return nil
 	}
 }
+
+// RawManifest returns kourier raw manifest without transformations
+func RawManifest(namespace string, apiclient client.Client) (mf.Manifest, error) {
+	return mfc.NewManifest(manifestPath(), apiclient)
+}
+
+// manifest returns kourier manifest after transformed
```

Comment:
This is not quite true. We should replace image references too if that's what we want here imo. Adding the new annotations doesn't hurt either.